### ISSUE TITLE
Are Movie fields scraper orders supposed to be editable?

### ISSUE THREAD

**Issue by wkearney99** [adultmm/AdultMediaManager#45]
Within settings for Movies, looking at the 'Movie fields' tab there are a bunch of fields using 'TMDB;AdultDVDEmpire;DorcelVision'

There's [...] box next to each field, but clicking it does nothing.  Is it supposed to?

Editing the AdvancedSettings.xml file doesn't work, as it gets re-written, canceling out any edits made externally.

It'd be convenient to change the search order a Movie is using there, or even to omit one or more of them.

**Comment by wkearney99** [comment]
![movie fields](https://user-images.githubusercontent.com/232313/169173387-6fd5f508-9ac9-42e3-8446-86b725d2389e.png)

**Comment by adultmm** [comment]
You're right, that button should do the work, but it's not implemented yet. 

In the [last beta (v0.11.0-BETA.1)](https://github.com/adultmm/AdultMediaManager/releases/tag/v0.11.0-BETA.1), you can set the enabled movie scrapers. Maybe, it can help you. 

I'll check the possibility to read the order from AdvancedSettings.xml, maybe it's not too much work to implement.

**Comment by wkearney99** [comment]
Yes, the BETA.1 has new features for selecting scrapers, nice!
